vld.props

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalOptions>/FI "vld.h"</AdditionalOptions>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);"c:\Program Files (x86)\Visual Leak Detector\include"</AdditionalIncludeDirectories>
+    </ClCompile>
+    <Link Condition="'$(Platform)' == 'x86'">
+      <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories);"c:\Program Files (x86)\Visual Leak Detector\lib\Win32"</AdditionalLibraryDirectories>
+    </Link>
+    <Link Condition="'$(Platform)' == 'x64'">
+      <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories);"c:\Program Files (x86)\Visual Leak Detector\lib\Win64"</AdditionalLibraryDirectories>
+    </Link>
+  </ItemDefinitionGroup>
+</Project>
```

Comment:
interesting, was this accidentally deleted?

Reply:
I think it was never moved to the PAL repo in the first place

---
In reply to: [516158253](https://github.com/Azure/c-pal/pull/53#discussion_r516158253) [](ancestors = 516158253)

Reply:
Annnd...it's gone. Taks to add it back because apparently the gate doesn't build with it #7768026

---
In reply to: [516187018](https://github.com/Azure/c-pal/pull/53#discussion_r516187018) [](ancestors = 516187018,516158253)